apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js

```diff
@@ -34,32 +35,36 @@ describe("DetailViewContents", () => {
     );
   };
 
-  it("Renders full contents initially", () => {
-    const detailView = mountDetailView();
+  for (const applicationType of ['Teacher1819Application', 'Facilitator1819Application']) {
+    it(`Renders full contents for ${applicationType} initially`, () => {
+      const detailView = mountDetailView(applicationType);
 
-    expect(detailView.find('#TopSection DetailViewResponse')).to.have.length(4);
-    expect(detailView.find('Facilitator1819Questions')).to.have.length(1);
-    expect(detailView.find('Facilitator1819Questions h3')).to.have.length(7);
-  });
+      expect(detailView.find('#TopSection DetailViewResponse')).to.have.length(4);
+      expect(detailView.find('DetailViewApplicationSpecificQuestions')).to.have.length(1);
+      expect(detailView.find('DetailViewApplicationSpecificQuestions h3')).to.have.length(
+        applicationType === 'Teacher1819Application' ? 4 : 7
+      );
```

Comment:
Rather than the ternary here, I'd rather this be more declarative and contain these numbers in an object.
Something like:
```jsx
const tests = [
  {type: 'Teacher1819Application`, applicationSpecificQuestions: 4},
  {type: 'Facilitator1819Application`, applicationSpecificQuestions: 7}
```

Then loop through that and refer to the relevant properties directly in the test.